app/code/Magento/Bundle/Helper/Model/Link.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Bundle\Helper\Model;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+class Link
+{
+    /**
+     * Clean extension attributes
+     *
+     * @param $selectionData
+     *
+     * @return array
+     */
+    public function cleanExtensionAttribute($selectionData): array
```

Comment:
Please add type hints.
```suggestion
    public function cleanExtensionAttribute(array $selectionData): array
```

Reply:
Has been updated